internal/notify/mouse.go

```diff
@@ -0,0 +1,6 @@
+package notify
+
+type MouseNotify interface {
+	MouseInNotify(x int, y int)
```

Comment:
As mentioned above, let's use fyne.Position as it's really just the wm package that uses x, y ints - this will help to differentiate between scaled and unscaled.